rebar.config

```diff
@@ -108,8 +108,7 @@
 
   %%% Other
   {pa, {git, "https://github.com/erszcz/pa.git", {branch, "master"}}},
-  %% MR of jwerl - https://gitlab.com/glejeune/jwerl/-/merge_requests/13
-  {jwerl, {git, "https://gitlab.com/vkatsuba/jwerl.git", {branch, "refactoring/otp-24"}}},
+  {jwerl, {git, "https://gitlab.com/glejeune/jwerl.git", {tag, "1.2.0"}}},
```

Comment:
A hex package was also published, better use that one: https://hex.pm/packages/jwerl